azure-pipelines.yml

```diff
@@ -278,13 +280,13 @@ stages:
         displayName: 'List dotnet versions and current version'
 
       - task: PowerShell@2
-        displayName: 'Run SonarAnalyzer integration tests'
+        displayName: 'Run ITs'
```

Comment:
pedantic: forgot about run here

Reply:
This one was intentional. We're inside `.NET ITs` job that has tasks (not visible in GitHub):

- Download binaries to test
- Install NuGet
- Install NET 5 SDK preview
- Install dotnet 3.1 LTS
- List dotnet versions and current version
- Run ITs

Considering this context, the last step should be named `Run` (or as I usually do when prototyping in non production code `GoGoGo` 🙂) because it's the actual step where the `.NET ITs` are triggered. 

I didn't want to repeat `.NET ITs` name and I didn't want to have just `Run` because it's too vague when you're looking just at the job.

Reply:
And I didn't want to use the `regression-test.ps1` in the name as well, that seems like an implementation detail.